pkg/environments/gitops.go

```diff
@@ -267,6 +345,36 @@ func CreateAddRequirementFn(chartName string, alias string, version string, repo
 	}
 }
 
+// CreateAddApplicationConfigFn create the ModifyAppsFn that adds an app to the ApplicationConfig
+func CreateAddApplicationConfigFn(chartName string, version string, repo string) ModifyAppsFn {
```

Comment:
I feel like this is the wrong name? "application", as best as I can tell, refers to things we've built and deployed, while "app" refers to the eternally-unsure-what-we're-supposed-to-be-calling-it external apps like `jx-app-ui` and `jx-app-sso`.

Reply:
@abayer good point. I'll rename the `jx-apps.yml` related config structs `AppConfig` rather than `ApplicationConfig` to avoid confusion